test.js

```diff
@@ -141,6 +141,40 @@ test('enqueue - add a message to the queue', t => {
   })
 })
 
+test('enqueue - stringify userId', t => {
+  const client = createClient()
+
+  client.track({
+    userId: 157963456373623802,
+    event: 'event'
+  }, noop)
+
+  t.is(client.queue.length, 1)
+
+  const item = client.queue.pop()
+
+  t.is(item.message.anonymousId, null)
+  t.is(typeof item.message.userId, 'string')
+  t.regex(item.message.userId, '157963456373623802')
```

Comment:
Should use `t.is` and delete the assertion above, because you're already expecting a string.

Reply:
Ah oops, just copy paste these from another test. Will update.

Reply:
Updated!